.github/workflows/ci.yml

```
@@ -108,12 +108,58 @@ jobs:
         env:
           RUSTFLAGS:
 
-  run-x86_64:
-    name: Run Hermit for Rust (x86_64)
+  run-hermit:
+    name: Run Hermit for Rust
```

```suggestion
    name: Run
```

If we shorten this, the matrix instances will be easier to on the overview: https://github.com/hermit-os/kernel/actions/runs/6434190933?pr=940